BUILD

```diff
@@ -15,6 +15,28 @@ exports_files([
 # @external_begin
 filegroup(
     name = "node_modules",
-    srcs = glob(["node_modules/**/*"]),
+    srcs = glob(
+        # Only include files we might use, to reduce the file count and surface size of 
+        # filename problems.
+        [
+            "node_modules/**/*.js",
+            "node_modules/**/*.json",
+            "node_modules/**/*.d.ts",
+        ],
+        exclude = [
+            # e.g. node_modules/adm-zip/test/assets/attributes_test/New folder/hidden.txt
+            "node_modules/**/test/**",
+            # e.g. node_modules/xpath/docs/function resolvers.md
+            "node_modules/**/docs/**",
+            # e.g. node_modules/puppeteer/.local-chromium/mac-536395/chrome-mac/Chromium.app/Contents/Versions/66.0.3347.0/Chromium Framework.framework/Chromium Framework
+            "node_modules/**/.*/**"
```

Comment:
we now also add an exclude for spaces
`node_modules/**/* *`

Reply:
Added that one too.